CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 ## Unreleased
+- Adds Tracing.getExportComponent().shutdown() for use within application shutdown hooks.
+- `Duration.create` now throws an `IllegalArgumentException` instead of
+  returning a zero `Duration` when the arguments are invalid.
+- `Timestamp.create` now throws an `IllegalArgumentException` instead of
+  returning a zero `Timestamp` when the arguments are invalid.
```

Comment:
Did you add these three entries while resolving a merge conflict?  They are not in the original commit.

Reply:
Removed https://github.com/census-instrumentation/opencensus-java/commit/b647dbda2e7131d7d350f63d28943c93fe94fc47, that one is less an issue and may be better put into next minor release.